client_check.sh

```diff
@@ -41,10 +41,12 @@ log="/tmp/${product}_client_check.log"
 echo -n > "${log}"
 
 if [ -f /etc/redhat-release ] || [ -f /etc/system-release ]; then
-  if [ "${product}" = "ps56" ] || [ "${product}" = "ps57" ]; then
+	if [ "${product}" = "ps56" ] || [ "${product}" = "ps57" ]; then
```

Comment:
Sorry it sometimes happens silently (VSCode shows proper indentation but after removal and reindentation it is detected by diff), I will check it using diff from now onwards to avoid its occurences.